src/CompanyService.Mappers/RequestMappers/Interfaces/IDepartmentMapper.cs

```diff
@@ -0,0 +1,14 @@
+﻿using LT.DigitalOffice.CompanyService.Mappers.Interfaces;
+using LT.DigitalOffice.CompanyService.Models.Db;
+using LT.DigitalOffice.CompanyService.Models.Dto.Requests;
+
+namespace LT.DigitalOffice.CompanyService.Mappers.RequestMappers.Interfaces
+{
+    /// <summary>
+    /// Represents mapper. Provides methods for converting an object of <see cref="NewDepartmentRequest"/>
+    /// type into an object of <see cref="DbDepartment"/> type according to some rule.
+    /// </summary>
+    public interface IDepartmentMapper : IMapper<NewDepartmentRequest, DbDepartment>
```

Comment:
Plz rename NewDepartmentRequest -> CreateDepartmentRequest

Reply:
I suggest to rename interface as well to IDbDepartmentMapper

Reply:
done